README.md

```diff
@@ -1 +1,35 @@
-"# awesome_books" 
+# Awesome Books
+
+> This project is an online library where you can add, find and remove books to display on screen.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
```

Comment:
[OPTIONAL] Kindly remove this section as there is no live demo, you add this only if there exists a live demo for the project